action.yml

```diff
@@ -7,6 +7,9 @@ inputs:
   site_name:
     description: "The Netlify site name to test against"
     required: true
+  extra_headers:
```

Comment:
`request_headers` might make a little more sense, since we aren't sending any headers currently.